pom.xml

```diff
@@ -251,6 +252,17 @@
                 <artifactId>maven-plugin-api</artifactId>
                 <version>${version.org.apache.maven.maven-core}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.maven.shared</groupId>
+                <artifactId>maven-artifact-transfer</artifactId>
+                <version>${version.org.apache.maven.shared}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
What is this for?

Reply:
For some classes required by featurePack class. As we can see in failing #276

Reply:
Sorry, I got the PR's mixed up :facepalm: